src/runtime/hexagon/hexagon/hexagon_buffer.cc

```diff
@@ -21,6 +21,10 @@
 
 #include <tvm/runtime/module.h>
 
+#ifdef BUILD_FOR_HEXAGON
```

Comment:
Had to stub this out for testing on CPU.  Not sure if this is the correct top level cmake switch to #ifdef around.